cmd/launcher/launcher.go

```diff
@@ -92,6 +93,11 @@ func main() {
 		logFatal(logger, "err", errors.Wrap(err, "detecting platform"))
 	}
 
+	debugTokenPath := filepath.Join(rootDirectory, "debug_token")
```

Comment:
I think the debug server should be hidden by a flag. Perhaps it's own flag, or perhaps just `--debug`.

Reply:
I'd like to reason about the possibility of leaving it on by default... We are not binding to any public address, and not exposing any particularly sensitive information. It's not as useful if someone encounters an issue and then cannot turn on the server for a running process.